var/spack/repos/builtin/packages/py-jmespath/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyJmespath(PythonPackage):
+    """JMESPath (pronounced "james path") allows you to declaratively
+       specify how to extract elements from a JSON document"""
+
+    homepage = "git://github.com/boto/jmespath"
```

Comment:
These links won't open in my browser, you should use the https version instead.

Reply:
Goodness, I am not sure where those came from. Yes, https much better.